metadata-io/src/main/java/com/linkedin/metadata/search/LineageSearchService.java

```diff
@@ -206,14 +218,18 @@ public LineageSearchResult searchAcrossLineage(
       }
     }
 
-    // set schemaField relationship entity to be its reference urn
-    LineageRelationshipArray updatedRelationships = convertSchemaFieldRelationships(lineageResult);
-    lineageResult.setRelationships(updatedRelationships);
+    if (SearchUtils.convertSchemaFieldToDataset(searchFlags)) {
```

Comment:
Is this flag still relevant?

Reply:
yeah this is just a helper method to keep the check constrained